main_project/view/task4.py

```diff
@@ -0,0 +1,9 @@
+"""below 3 commands are used to fetch
+1)enroll student into particular course
+2)count of student enrolled into particular course
+3)count distinct students taught by single professor
+"""
+from main_project.controller.enrollment import student_enrollment_course,get_student_count_enrolled_into_course,get_distinct_students
+# student_enrollment_course()
```

Comment:
lets add comment to explain why this function is called here and uncomment the function calls